runtime/altair/src/lib.rs

```diff
@@ -173,7 +173,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("altair"),
 	impl_name: create_runtime_str!("altair"),
 	authoring_version: 1,
-	spec_version: 1103,
+	spec_version: 1200,
```

Comment:
FYI: Bumping to 1200 here and removing outdated migrations so I can use this as RC for Altair.